scripts/remote_diff/utils.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+
+
+def get_zone(project, host):
+    zone_call = subprocess.run(
+        [
+            "gcloud compute instances list \
+        --project={} | grep ^{} | awk -F ' ' '{{print $2}}'".format(
+                project, host)
+        ],
+        stdout=subprocess.PIPE,
+        check=True,
+        text=True,
+        shell=True,
+    )
+    return zone_call.stdout.replace('\n', '')
+
+
+def run_on_machine(command, user, host, project):
+    zone = get_zone(project, host)
+    call = subprocess.run(
+        [
+            "gcloud compute ssh {}@{} --command='{}' --project {} --zone {} ".
```

Comment:
Maybe use f-strings to make it easier for the readers?

```suggestion
            f"gcloud compute ssh {user}@{host} --command '{command}' --project {project} --zone {zone}"
```